sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -225,24 +225,21 @@ namespace Azure { namespace Security { namespace Attestation {
     /**
      * @brief Perform a single leg
      *
-     * Processes attestation evidence from a VBS enclave, producing an attestation result.
+     * Attest a TPM based enclave
      *
      * The TPM attestation protocol is defined
```

Comment:
QQ: The documentation in the "virtualization-based-security-protocol" is incorrect (it leaves off the "payload" JSON object). Should that mistake be reflected in the documentation for the `AttestTPM` API, or should it be fixed on the protocol documentation?

Reply:
We will address the issue in the documentation. Will create a task to track it